Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -148,7 +148,11 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     }
 
     private var isKeyExpired: Bool {
-        keyStore.currentKeyPair().expirationDate <= Date()
```

Comment:
This was creating a new keypair, but not registering it with the backend.

Reply:
🤯 Amazing catch